kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/streamable/StreamableKafkaMessageSource.java

```diff
@@ -120,9 +119,9 @@ public BlockingStream<TrackedEventMessage<?>> openStream(TrackingToken trackingT
         KafkaTrackingToken token = KafkaTrackingToken.from(trackingToken);
         TrackingRecordConverter<K, V> recordConverter = new TrackingRecordConverter<>(messageConverter, token);
 
-        logger.debug("Will start consuming from topics [{}]", topics);
+        logger.debug("Will start consuming from topics [{}]", subscriber);
```

Comment:
I am not sure whether this is helpful as a log statement, as the `KafkaSubscriber` does not have a `describer()` kinda method returning the topics or topic pattern. Might be a nice addition to the interface actually. What do you think?

Reply:
Added, let me know your thoughts :)

Reply:
Looks good to me! :-) Thanks for the addition!